File: sdkwork-backend-sdk-csharp/Models/AecConfig.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class AecConfig
    {
        public int? EchoMode { get; set; }
        public int? DelayEstimation { get; set; }
        public int? NoiseSuppression { get; set; }
        public int? EchoTailMs { get; set; }
        public int? SampleRate { get; set; }
    }
}

File: sdkwork-backend-sdk-go/types/aec_config.go
package types


type AecConfig struct {
	EchoMode int `json:"echoMode"`
	DelayEstimation int `json:"delayEstimation"`
	NoiseSuppression int `json:"noiseSuppression"`
	EchoTailMs int `json:"echoTailMs"`
	SampleRate int `json:"sampleRate"`
}
